aca_preview/preview.py

```diff
@@ -32,6 +32,7 @@
 PROSECO_VERSION = proseco.test(get_version=True)
 FILEDIR = Path(__file__).parent
 
+GAIA_BAD_PM = Table.read('/proj/sot/ska/analysis/gaia/agasc_gaia_xmatch_PM_gt_50mas.fits.gz')
```

Comment:
It would probably be fine to just put the text list of bad stars in here for now as well.  Until we do https://github.com/sot/ska-projects/issues/92

Reply:
Add a comment to see the `check_bad_agasc_proper_motion` function for more details.